kernel-modules/container/required-layers.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+import re
+
+probe_name_re = re.compile(r'^collector.*.gz$')
+
+
+def get_probe_sizes(probe_dir):
```

Comment:
[nit] Type hinting would be nice here, quickly glancing through the code I initially thought this method returned the added size of all probes instead of the individual sizes in an array, (I know I should've paid more attention, but still).
```suggestion
def get_probe_sizes(probe_dir: str) -> list:
```